tests/contract/test_resource_client.py

```diff
@@ -202,6 +215,51 @@ def resource_client_inputs_schema():
     return client
 
 
+@pytest.fixture
+def resource_client_inputs_additional_identifiers():
+    endpoint = "https://"
+    patch_sesh = patch(
+        "rpdk.core.contract.resource_client.create_sdk_session", autospec=True
+    )
+    patch_creds = patch(
+        "rpdk.core.contract.resource_client.get_temporary_credentials",
+        autospec=True,
+        return_value={},
+    )
+    patch_account = patch(
+        "rpdk.core.contract.resource_client.get_account",
+        autospec=True,
+        return_value=ACCOUNT,
+    )
+    with patch_sesh as mock_create_sesh, patch_creds as mock_creds:
+        with patch_account as mock_account:
+            mock_sesh = mock_create_sesh.return_value
+            mock_sesh.region_name = DEFAULT_REGION
+            client = ResourceClient(
+                DEFAULT_FUNCTION,
+                endpoint,
+                DEFAULT_REGION,
+                SCHEMA_WITH_ADDITIONAL_IDENTIFIERS,
+                EMPTY_OVERRIDE,
+                {
+                    "CREATE": {"a": 111, "c": 2, "d": 3},
+                    "UPDATE": {"a": 1, "c": 2},
+                    "INVALID": {"c": 3},
+                },
+            )
+
+    mock_sesh.client.assert_called_once_with("lambda", endpoint_url=endpoint)
+    mock_creds.assert_called_once_with(mock_sesh, LOWER_CAMEL_CRED_KEYS, None)
```

Comment:
Can we reuse an existing resource client vs creating a new one?

Reply:
Sure, I was following the style of the other tests at the time but it seems like that has been changed since. I'll update to the new style.

Reply:
I have addressed this by parameterizing the fixture, let me know if that works for you.